packages/ui/src/onboarding/Login/pages/WorkspaceLogin/index.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Paper,
+  createStyles,
+  TextInput,
+  Button,
+  Text,
+  Anchor,
+  Divider,
+  Flex
+} from '@mantine/core';
+import FormTitle from '../../components/FormTitle';
+
+const useStyles = createStyles((theme) => ({
+    loginPage: {
+        height: '100vh',
+        background: '#FCF7F1',
+    },
+    
+    wrapper: {
+        width: '100%',
+        height: '100%',
+        minHeight: '100vh',
+    },
+
+    formContainer: {
+        width: '60%',
+        height: '100%',
+        positon: 'relative',
+        padding: '17px',
+        [`@media (max-width: ${theme.breakpoints.sm}px)`]: {
+            width: '100%',
+        },
+    },
+
+    formHeader: {
+        position: 'fixed',
+        left: '0',
+        right: '0',
+        top: '0',
+        height: '75px',
+        borderBottom: '1px solid #F9DBB6',
+        display: 'flex',
+        justifyContent: 'start',
+        alignItems: 'center',
+        paddingLeft: '110px',
+        [`@media (max-width: ${theme.breakpoints.sm}px)`]: {
+            paddingLeft: '17px',
+        },
+    },
+
+    formLogo: {
+        display: 'flex',
+        gap: '10px',
+        justifyContent: 'start',
+        alignItems: 'center',
+        
+    },
+
+    form: {
+        // borderRight: `1px solid ${
+        // theme.colorScheme === 'dark' ? theme.colors.dark[7] : theme.colors.gray[3]
```

Comment:
remove all commented

only leave comments that give a description or provide context on why something was done